shell/src/main/java/org/apache/accumulo/shell/commands/ScriptCommand.java

```diff
@@ -197,7 +206,7 @@ private void putConnector(Bindings b, AccumuloClient client) {
 
   @Override
   public String description() {
-    return "execute JSR-223 scripts";
+    return "This is will be deprecated and will be replaced by Java 11's JShell Feature - execute JSR-223 scripts";
```

Comment:
If I remember correctly, these command descriptions are used by the Shell to generate it's built-in `help` documentation. Therefore, the description style should flow well in that context. Have you tried running the `bin/accumulo shell` to test this description in the help context?

Reply:
I have reverted the comment to as before, We have deprecated the class and eventually they will be removed when we move to Jshell, I didn't see any value adding it to the description.